python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -883,6 +931,8 @@ def startup(self):
             self.repository.startup()
             # All Ids could have changed
             self.changed_ids = {}
+            self.flush_thread = RegistryFlusher(self)
+            self.flush_thread.start()
```

Comment:
Do all Registries call the baseclass for ```startup```?
Is it worth having this configurable within each registry?